dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -155,7 +156,12 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
             //TODO: would hook up identified tools that form workflows here
             toolVersion.setIncludedApps(MoreObjects.firstNonNull(toolVersion.getIncludedApps(), Lists.newArrayList()));
             //TODO: hook up snapshot and checksum behaviour here
-            toolVersion.setIsProduction(false);
```

Comment:
I think you should restore this line, change it to `toolVersion.setIsProduction(version.isFrozen())`, then get rid of the else on line 162.